matching-core/src/main/java/com/graphhopper/matching/MapMatching.java

```diff
@@ -65,10 +62,17 @@
  */
 public class MapMatching {
 
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    // Penalty in m for each U-turn performed in the path between two subsequent candidates
+    // Note that this penalty should roughly correspond to the penalty used for unfavored
+    // virtual edges because results get inconsistent otherwise.
+    private static final double VIRTUAL_NODE_U_TURN_PENALTY = 100;
+
     private final Graph routingGraph;
     private final LocationIndexMatch locationIndex;
     private double measurementErrorSigma = 50.0;
-    private double transitionProbabilityBeta = 0.00959442;
+    private double transitionProbabilityBeta = 2.0;
```

Comment:
This parameter needed to be adjusted because because the normalized transition metric was changed to the non-normalized transition metric. This was necessary because the normalized transition metric divides by the squared time difference but for some traces such as #13, all time stamps are equal. The previous way to deal with this doesn't work with directed candidates and also has other problems as explained [above](https://github.com/graphhopper/map-matching/pull/88#discussion_r92124497).